src/Console/Command/ReleaseCommand.php

```diff
@@ -0,0 +1,289 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DevKit\Console\Command;
+
+use Github\Exception\ExceptionInterface;
+use Packagist\Api\Result\Package;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Jordi Sala <jordism91@gmail.com>
+ */
+final class ReleaseCommand extends AbstractCommand
+{
+    private static $labels = array(
+        'patch' => 'blue',
+        'bug' => 'red',
+        'docs' => 'yellow',
+        'minor' => 'green',
+        'pedantic' => 'cyan',
+    );
+
+    private static $stabilities = array(
+        'patch' => 'blue',
+        'minor' => 'green',
+        'stable' => 'yellow',
+    );
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $this
+            ->setName('release')
+            ->setDescription('Makes a release of a project.')
+            ->addArgument('project', InputArgument::REQUIRED, 'The project to do a release of.')
```

Comment:
Added, my english is pretty bad, but @greg0ire could help improving the text :P .